src/site/content/en/blog/periodic-background-sync/index.md

```diff
@@ -315,13 +314,13 @@ Sync API.
   syncs. Retrieve an instance of this class from the ServiceWorkerRegistration.periodicSync`
   property.
 * `ServiceWorkerGlobalScope.onperiodicsync`. Registers a handler to receive the `PeriodicSyncEvent`.
-* `ServiceWorkerRegistration.periodicSync`. Returns a reference to the `PeriodicSyncManager`.
+* `ServiceWorkerRegistration.periodicSync`. Holds a reference to the `PeriodicSyncManager`.
```

Comment:
```suggestion
* `ServiceWorkerRegistration.periodicSync`. Returns a reference to the `PeriodicSyncManager`.
```

Reply:
I guess 'holds' would be more logical, but 'returns' is [what's typically done](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration#Properties). Language follows convention more than it does strict logic.

Reply:
'returns' implies that it's a function rather than a property to me, but I'm happy to go with convention here.

Reply:
I think you should. 

This is why I'm so strict about including `()` at the end of function names.